oneanddone/tasks/templates/tasks/activity_listing.html

```diff
@@ -24,6 +24,12 @@
           <div>{{ _("Team: {team}")|fe(team=attempt.task.team.name) }}</div>
           <div>{{ _('Date {state}: {modified}')|fe(state=attempt.get_state_display(),
             modified=attempt.modified.strftime('%Y-%m-%d: %I:%M:%S %p').lower()) }}</div>
+          <div>
+            {% if attempt.has_feedback %}
```

Comment:
We need to check not only that the attempt has feedback, but also that `time_spent_in_minutes` is not `None`, because there will be old `Feedback` entries that do not have a value for `time_spent_in_minutes`.